src/main/java/cn/nukkit/Player.java

```diff
@@ -1830,7 +1830,7 @@ protected void checkBlockCollision() {
                 } else {
                     Level end = this.getServer().getLevelByName("the_end");
                     if (end != null) {
-                        this.teleport(end.getSafeSpawn(), TeleportCause.END_PORTAL);
+                        this.teleport(new Position(100.5, 49, 0.5, end), TeleportCause.END_PORTAL);

```

Comment:
I think this will invalidate the spawn point setting of the end world
I think we should change the default configuration of end world generation